### DISCUSSION TITLE
How to make use of composition properties ?

### DISCUSSION THREAD

**Discussion by Kisis-4SH** [teambit/bit#6721] [Help]
I can see there are "properties" for compositions.
But I don't see anything about this on the docs, and it doesn't seem that we can put args to the compositions functions like StoryBook's Stories as below
```ts
interface CompositionArgs {
    toto: string
}

export const Composition = ({toto}: CompositionArgs) => <l-field name="World" placeholder={toto}></l-field>;
Composition.args = {
    toto: 'toto'
}
```

So, how do we use them ?

**Comment by ranm8** [comment]
Bit does not support visual controls and property setting in compositions. I would recommend to use Storybook if these capabilities are required. Learn how to [use Storybook in your Bit workspace](https://bit.cloud/bitdesign/storybook/storybook).
